pkgs/shells/oh-my-zsh/default.nix

```diff
@@ -64,7 +64,7 @@ stdenv.mkDerivation rec {
   To copy the Oh My Zsh configuration file to your home directory, run
   the following command:
 
-    $ cp -v $(nix-env -q --out-path oh-my-zsh-git | cut -d' ' -f3)/share/oh-my-zsh/templates/zshrc.zsh-template ~/.zshrc
+    $ cp -v $(nix-env -q --out-path oh-my-zsh | cut -d' ' -f3)/share/oh-my-zsh/templates/zshrc.zsh-template ~/.zshrc
```

Comment:
Ah, that looks very good! Actually, better than then having a separate `oh-my-zsh` package.

@NeQuissimus, what do you think?  Should we drop this package in favour of the module?